src/compounds/Footer/Footer2.stories.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
```

Comment:
I'm curious about whether we can refactor this file into "Footer.stories.jsx"...but we can save that for another day. 🙂

Reply:
ooooh thats a good point! i think that could potentially be doable